web/portal/platform/src/components/Login.tsx

```diff
@@ -2,6 +2,8 @@ import { useEffect } from 'react';
 import { useStoreActions, useStoreState } from 'store';
 import { EntityValidator } from '@irontec/ivoz-ui/entities/EntityInterface';
 import { Login as DefaultLogin } from '@irontec/ivoz-ui/components';
+import queryString from 'query-string';
```

Comment:
Revert this file, platform is the highest level portal. No one can exchange token here